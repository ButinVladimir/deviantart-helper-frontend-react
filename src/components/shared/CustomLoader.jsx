import React from 'react';
import Section from 'react-bulma-components/lib/components/section';
import Container from 'react-bulma-components/lib/components/container';
import Level from 'react-bulma-components/lib/components/level';
import Loader from 'react-bulma-components/lib/components/loader';

export default function CustomLoader() {
  return (
    <Section>
      <Container>
        <Level>
          <Level.Item>
            <Loader className="custom-loader" />
          </Level.Item>
        </Level>
      </Container>
    </Section>
  );
}
