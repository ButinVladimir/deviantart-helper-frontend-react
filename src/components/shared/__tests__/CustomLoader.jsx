import React from 'react';
import { shallow } from 'enzyme';
import CustomLoader from '../CustomLoader';

describe('CustomLoader', () => {
  it('can be rendered correctly', () => {
    const wrapper = shallow(<CustomLoader />);

    expect(wrapper).toMatchSnapshot();
  });
});
