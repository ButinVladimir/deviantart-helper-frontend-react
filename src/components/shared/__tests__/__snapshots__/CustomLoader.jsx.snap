// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomLoader can be rendered correctly 1`] = `
<Section
  className=""
  renderAs="section"
  size={null}
  style={Object {}}
>
  <Container
    breakpoint={null}
    fluid={false}
    renderAs="div"
    style={Object {}}
  >
    <Level
      breakpoint={null}
      className=""
      renderAs="div"
      style={Object {}}
    >
      <LevelItem
        className=""
        renderAs="div"
        style={Object {}}
      >
        <Loader
          className="custom-loader"
          renderAs="div"
          style={Object {}}
        />
      </LevelItem>
    </Level>
  </Container>
</Section>
`;
